src/languages/ruby.js

```diff
@@ -120,19 +114,16 @@ export default function(hljs) {
     className: 'subst',
     begin: /#\{/,
     end: /\}/,
-    keywords: RUBY_KEYWORDS
+    keywords: RUBY_KEYWORDS,
+    relevance: 10
```

Comment:
Please do not relevance spike.  Relevance is going away entirely in v12.

Reply:
I've set this to 2

Reply:
Please remove all relevance changes you've introduced.  You can leave what is already there... this should reduce merge conflicts with my other relevance changes.

Reply:
removed